src/main/kotlin/no/nav/syfo/aktivitetskrav/cronjob/JournalforAktivitetskravVarselCronjob.kt

```diff
@@ -37,7 +38,8 @@ class JournalforAktivitetskravVarselCronjob(
                 val journalpostRequest = createJournalpostRequest(
                     personIdent = personIdent,
                     navn = navn,
-                    pdf = pdf
+                    pdf = pdf,
+                    varselId = varsel.uuid,
```

Comment:
Burde vi kalle parameteren `varselUuid`? Kan virke som det er en database-id hvis vi bare bruker Id.

Reply:
Det er dårlig ddd å kalle en variabel for `uuid` da 🤷‍♂️